rust/utils/run-locally/src/main.rs

```diff
@@ -142,23 +142,29 @@ fn main() -> ExitCode {
     };
 
     let relayer_env = hashmap! {
```

Comment:
No net changes to the config, just trying to make sure the args correctly are parsed and override the env.